### DISCUSSION TITLE
Arcs still enabled after disabling

### DISCUSSION THREAD

**Discussion by jlmurray4279** [OrcaSlicer/OrcaSlicer#4054] [General]
Ill look into it a bit more but I disable arcs and set the slicer language to klipper. However I believe on layer changes, I am seeing in the klipper console unknown command G3 and G2. It seems to be layer changes I suspect that it is retraction is set to auto and doing a spiral retraction. Enabling arc in klipper would fix this issue. Has anyone else had this issue? Maybe someone already pin pointed the problem

**Comment by igiannakas** [comment]
It's the zhop. If set to auto it will do spiral Z that uses G2/3 commands. Set it to normal or slope and the arc commands should disappear.

**Reply by trebory6** [reply]
So what should I do if I actually want to use Spiral since it says it works better that way?

Can I enable those commands on Klipper to properly execute these gcode commands?

**Reply by igiannakas** [reply]
If you want to use spiral Z hop enable arcs in klipper and set the zhop mode to auto or spiral. You do not need to enable arc fitting in the quality tab though. This would affect the model slicing. The zhop arcs are independent from this.

**Reply by pm0u** [reply]
it would be awesome if at the very least the hover tooltips noted that auto z hop or spiral will lead to arcs.

**Reply by igiannakas** [reply]
It doesn’t any more.

**Reply by pm0u** [reply]
As of what version? I just ran into this on 2.3.1

**Reply by pm0u** [reply]
Think i found the PR https://github.com/OrcaSlicer/OrcaSlicer/pull/11430

Which looks to be after 2.3.1 - glad to see this resolved but I guess worth noting for accuracy/reducing confusion this will be fixed in the 2.3.2 release which is not stable yet.
